app/models/reports/schema_v2.rb

```diff
@@ -226,5 +186,38 @@ def percentage(numerator, denominator)
       return 0 if numerator.nil? || denominator.nil? || denominator == 0 || numerator == 0
       ((numerator.to_f / denominator) * 100).round(PERCENTAGE_PRECISION)
     end
+
+    # Grab a particular summed field for a region. We return data in the format of:
+    #   { region_slug => { period_1 => x, period_2 => y }, ...}
+    # to maintain consistency w/ the format callers expect from the Repository.
+    #
+    # Note that we do filtering on the result set to limit the returned amount of data to the data that callers are
+    # requesting via the `periods` argument the Repository was created with.
+    def sum(region, field)
+      summed_field = "sum_#{field}"
```

Comment:
We use this in a couple of places and it feels like an important implementation detail. Can we extract a function for it?

Reply:
👍